core/packages/test/package.json

```diff
@@ -14,8 +14,10 @@
   "devDependencies": {
     "@chainsafe/lodestar": "1.1.0",
     "@polkadot/api": "^9.6.2",
+    "@polkadot/types": "^9.6.2",
     "@polkadot/util": "^10.1.11",
     "@polkadot/util-crypto": "^10.1.11",
+    "@snowfork/snowbridge-types": "^0.2.7",
```

Comment:
This package is obsolete, we shouldn't rely on it anymore. I see you're adding it because you included `make-xcm-transfer.ts` in the compilation. But that script is obsolete too, so we should probably exclude it from compilation until its replaced.